src/main/kotlin/edu/ycp/cs482/iorcapi/factories/CharacterFactory.kt

```diff
@@ -53,6 +53,23 @@ class CharacterFactory(
 
         val abils = Ability(abilityPoints.str, abilityPoints.con, abilityPoints.dex, abilityPoints._int, abilityPoints.wis, abilityPoints.cha)
 
+
+        val charNew = Character(id,
+                name = name,
+                abilityPoints = abils,
+                raceid = race.id,
+                classid = classql.id,
+                version = char.version,
+                inventory = char.inventory,
+                slots = updateSlotsIfEmpty(char),
+                money = char.money
+            )
+        characterRepo.save(charNew) //should this be insert?
```

Comment:
if this question is for future reference try not to forget about it, other than that everything seems fine

Reply:
That question is on all .save() rn. It's part of an ongoing issue #16, mostly, its wanted behavior. I should take out those comments.